integration_tests/experiments/memcached-sensitivity-profile/experiment_test.go

```diff
@@ -157,9 +157,9 @@ func TestExperiment(t *testing.T) {
 			experimentID, err := runExp(memcachedSensitivityProfileBin, true, "-experiment_be_workloads", sensitivity.NoneAggressorID)
 
 			Convey("Experiment should return with no errors and there should be 9 metrics in Cassandra", func() {
+				So(err, ShouldBeNil)
```

Comment:
@skonefal - it has always been in separate `Convey()` - event its name indicates it.